src/lib.rs

```diff
@@ -241,21 +248,26 @@ mod hwb;
 
 mod hues;
 
-mod tristimulus;
 mod convert;
 mod equality;
+pub mod chromatic_adaptation;
+pub mod white_point;
+pub mod matrix;
+pub mod rgb_primaries;
```

Comment:
I'm not sure if we should make this module public right now. It's basically an implementation detail at this stage.